windows/hid.c

```diff
@@ -183,8 +185,28 @@ struct hid_device_ {
 		OVERLAPPED ol;
 		OVERLAPPED write_ol;
 		struct hid_device_info* device_info;
+		BOOL use_hid_write_output_report;
 };
 
+static BOOL IsWindowsVersionOrGreater(WORD wMajorVersion, WORD wMinorVersion, WORD wServicePackMajor)
```

Comment:
I'd prefer `hid_internal_is_windows_version_or_greater` - so it is easily distinguishable from public method